src/main/groovy/org/shipkit/gradle/ReleaseNeededTask.java

```diff
@@ -24,23 +19,15 @@
  *  - the current Git branch does not match release-eligibility regex ({@link #getReleasableBranchRegex()}.
  *  - binaries have not changes since the previous release
  */
+//TODO move to 'release' subpackage
 public class ReleaseNeededTask extends DefaultTask {
```

Comment:
This is a great idea to use Gradle's build cache! Thanks!!!

We will probably use programatic API instead of the annotation to avoid compile time dependency on latest Gradle. This also allows us to conditionally apply the setting based on the Gradle version. Also, we need to add it to more tasks, most of our tasks are incremental :)

Thanks for suggestion and the review!